pkg/arvo/app/hark-store.hoon

```diff
@@ -596,6 +598,16 @@
     =>  (emit autoseen-timer)
     poke-core(current-timebox now.bowl)
   ::
+  ++  read-all
+    =.  unreads-count
+      %-  ~(run by unreads-count)
+      |*(* 0)
+    =.  poke-core  seen
+    %-  ~(rep ^in ~(key by unreads-each))
```

Comment:
I think this would look clearer as a `=:`

Reply:
(As in use the tiscol to reset unreads-count and unreads-each, and then just return `seen` as the last subhoon)